lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/MavenLemminxExtension.java

```diff
@@ -467,4 +471,13 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
 
 		return projectsToAdd;
 	}
+	
+	private void registerCodeActionParticipants(XMLExtensionsRegistry registry) {
+		registry.registerCodeActionParticipant(new MavenNoGrammarConstraintsCodeAction());
+	}
+	
+	private void unregisterCodeActionParticipants(XMLExtensionsRegistry registry) {
+		registry.getCodeActionsParticipants().stream().collect(Collectors.toList())
+			.forEach(p -> registry.unregisterCodeActionParticipant(p));

```

Comment:
```suggestion
			.forEach(registry::unregisterCodeActionParticipant);
```
saves a few bytes by using a method reference instead of a lambda.